packages/bits/spec/components/textbox-number/textbox-number.atom.ts

```diff
@@ -40,4 +39,6 @@ export class TextboxNumberAtom extends Atom {
 
     public isValid = async (): Promise<boolean> => !(await this.rootElement.getAttribute("class")).includes("has-error");
 
+    public getInputId = async (): Promise<string> => await this.input.getId();
```

Comment:
It might be better to just make the `input` property public so that consumers can access the input's ElementFinder api directly. That way, we don't have to expand the API of this atom every time the consumer needs to access something else on the input element, and this `getInputId` method can just be removed.

Reply:
Yes, you are right, it's a good point. Thank you